github_changelog_generator.gemspec

```diff
@@ -25,6 +25,7 @@ Gem::Specification.new do |spec|
 
   spec.add_runtime_dependency("github_api", ["~> 0.12"])
   spec.add_runtime_dependency("colorize", ["~> 0.7"])
+  spec.add_runtime_dependency("sqlite3")
```

Comment:
(It'll only load as jruby-dbm if we make it to.)

(Could we turn things upside-down - have a new problem - try having VCR help us do the request caching?)